monitoring/uss_qualifier/scenarios/astm/netrid/display_data_evaluator.py

```diff
@@ -369,6 +369,33 @@ def _evaluate_normal_observation(
                             details=f"{mapping.injected_flight.uss_participant_id}'s flight with injection ID {mapping.injected_flight.flight.injection_id} in test {mapping.injected_flight.test_id} had telemetry index {mapping.telemetry_index} at {injected_telemetry.timestamp} with lat={injected_telemetry.position.lat}, lng={injected_telemetry.position.lng}, alt={injected_telemetry.position.alt}, but {observer.participant_id} observed lat={observed_position.lat}, lng={observed_position.lng}, alt={observed_position.alt} at {query.request.initiated_at}",
                         )
 
+        # Check that flights using telemetry are not using extrapolated position data
+        for mapping in mapping_by_injection_id.values():
+            injected_telemetry = mapping.injected_flight.flight.telemetry[
+                mapping.telemetry_index
+            ]
+
+            with self._test_scenario.check(
+                "Telemetry being used when present",
+                [
+                    observer.participant_id,
+                    mapping.injected_flight.uss_participant_id,
+                ],
+            ) as check:
+                if (
+                    "extrapolated" not in injected_telemetry.position
+                    or injected_telemetry.position["extrapolated"] is False
```

Comment:
Python's `is` is a reference comparison.  It might work here, but we already have a boolean so we should evaluate directly with it.

```suggestion
                    or not injected_telemetry.position["extrapolated"]
```